tests/data/pings.yaml

```diff
@@ -66,3 +66,20 @@ custom-ping-no-info:
   metadata:
     include_info_sections: false
   no_lint: [REDUNDANT_PING]
+
+custom-with-ping-schedule:
+  description:
+    A custom ping with a ping schedule
+  include_client_id: true
```

Comment:
You should be able to omit the `include_client_id: true`. This parameter is optional and it defaults to `true` already. Although, I do see this occurs in other places so feel free to disregard this nit.

Reply:
If I remove that line, I get these test failures:

```
FAILED tests/test_javascript.py::test_ping_parser - AssertionError: assert set() == {'pings.js'}
FAILED tests/test_javascript.py::test_reasons - AssertionError: assert set() == {'pings.js', 'pings.ts'}
FAILED tests/test_kotlin.py::test_ping_parser - AssertionError: assert set() == {'GleanBuildI...', 'Pings.kt'}
FAILED tests/test_kotlin.py::test_reasons - AssertionError: assert set() == {'GleanBuildI...', 'Pings.kt'}
FAILED tests/test_lint.py::test_unknown_pings_lint - AssertionError: assert 1 == 0
FAILED tests/test_markdown.py::test_reasons - AssertionError: assert set() == {'metrics.md'}
FAILED tests/test_markdown.py::test_send_if_empty_metrics - AssertionError: assert set() == {'metrics.md'}
FAILED tests/test_parser.py::test_parser - AssertionError: assert 1 == 0
FAILED tests/test_parser.py::test_ping_ordering - AssertionError: assert 1 == 0
FAILED tests/test_rust.py::test_ping_parser - AssertionError: assert set() == {'glean_metrics.rs'}
FAILED tests/test_swift.py::test_ping_parser - AssertionError: assert set() == {'Metrics.swift'}
FAILED tests/test_swift.py::test_reasons - AssertionError: assert set() == {'Metrics.swift'}
```

Not entirely sure why that is, so since the other pings in this file have this, I'll just leave this for now.